tests/testClassifier.py

```diff
@@ -48,17 +51,13 @@ def test1(self):
         self.assertIn("063.000.000", self.records)
         entries, total, updates = classifier.classify(self.records["063.000.000"])
 
-        # Condor doesn't emit the the current year in records.  The classifier
-        # code assumes it's the current year, and prepends that, so we have
-        # to assume that here as well when testing for the date.
-        year = str(date.today().year)
         rec = entries[0]
         self.assertEqual(rec.condorId, "063.000.000")
         self.assertEqual(rec.dagNode, "A2")
-        self.assertEqual(rec.submitTime, year+"-10-17 19:59:57")
+        self.assertTrue(rec.submitTime.endswith("-10-17 19:59:57"))
```

Comment:
I changed this to assertRegexpMatches initially, but then realized that because of the nature of the fix in this overall ticket, we now can check the entire date string rather than just part of it.  Since we know what the actual string should be, this reverts to an assertEqual in these tests.  A year to year job records unit test was added as well, which tests the change in dates.